pricegraph/src/lib.rs

```diff
@@ -249,20 +293,17 @@ mod tests {
             ],
         };
 
-        assert_eq!(
-            transitive_orderbook.ask_prices().collect::<Vec<_>>(),
-            vec![
-                (0.5 / FEE_FACTOR, 20_000_000.0),
-                ((0.9 / 1.5) / FEE_FACTOR, 1_500_000.0)
-            ]
-        );
-        assert_eq!(
-            transitive_orderbook.bid_prices().collect::<Vec<_>>(),
-            vec![
-                (0.5 * FEE_FACTOR, 2_000_000.0),
-                ((1.0 / 1.8) * FEE_FACTOR, 900_000.0)
-            ]
-        );
+        let ask_prices = transitive_orderbook.ask_prices().collect::<Vec<_>>();
+        assert_approx_eq!(ask_prices[0].0, 2.0 * FEE_FACTOR);
+        assert_approx_eq!(ask_prices[0].1, 10_000_000.0);
+        assert_approx_eq!(ask_prices[1].0, 1.5 / 0.9 * FEE_FACTOR);
```

Comment:
nit: I would prefer if the order of operations were clarified to remove some ambiguity when reading the code.
```suggestion
        assert_approx_eq!(ask_prices[1].0, (1.5 / 0.9) * FEE_FACTOR);
```